sympy/stats/joint_rv_types.py

```diff
@@ -394,6 +394,100 @@ def MultivariateEwens(syms, n, theta):
+        _value_check((v > 0) != False, "v must be positive")
+        for lk in l:
+            _value_check((lk > 0) != False, "lamda must be a positive vector.")
+        for muk in mu:
+            _value_check((muk > 0) != False, "mu must be a positive vector.")
+        _value_check(omega.shape == (len(l), len(mu)),
+                        "lamda, mu should be of same length and omega should "
+                        " be of shape (length of lamda, length of mu)")
+
+    @property
+    def set(self):
+        from sympy.sets.sets import Interval
+        return S.Reals**len(self.l)
+
+    def pdf(self, *y):
+        from sympy.functions.special.gamma_functions import gamma
+        omega, v, l, mu = self.omega, self.v, self.l, self.mu
+        n = Symbol('n', negative=False, integer=True)
+        k = len(l)
+        d = Pow(Rational(omega.det()), Rational(1, k - 1))
+        sterm1 = Pow((1 - d), n)/\
+                ((gamma(v + n)**(k - 1))*gamma(v)*gamma(n + 1))
+        sterm2 = Mul.fromiter([mui*li**(-v - n) for mui, li in zip(mu, l)])
+        term1 = sterm1 * sterm2
+        sterm3 = (v + n) * sum([mui * yi for mui, yi in zip(mu, y)])
+        sterm4 = sum([exp(mui * yi)/li for (mui, yi, li) in zip(mu, y, l)])
+        term2 = exp(sterm3 - sterm4)
+        return Pow(d, v) * Sum(term1 * term2, (n, 0, S.Infinity))
+
+def GeneralizedMultivariateLogGamma(syms, omega, v, l, mu):
+    """
+    Creates a joint random variable with generalized multivariate log gamma
+    distribution.
+
+    The joint pdf can be found at [1].
+
+    Parameters
+    ==========
+
+    syms: list/tuple/set of symbols for identifying each component
+    omega: A matrix
+           with each square root of absolute value of correlation
+    v: positive real
+    l: a list of positive reals
+    mu: a list of positive reals
+
+    Returns
+    =======
+
+    A Random Symbol
+
+    Examples
+    ========
+
+    >>> from sympy.stats import density
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> from sympy.stats.joint_rv_types import GMVLG
+    >>> from sympy import Matrix, symbols
+    >>> omega = Matrix([[1, 0.5, 0.5], [0.5, 1, 0.5], [0.5, 0.5, 1]])
+    >>> v = 1
+    >>> l, mu = [1, 1, 1], [1, 1, 1]
+    >>> G = GMVLG('G', omega, v, l, mu)
+    >>> y = symbols('y_1:4', positive=True)
+    >>> density(G)(y[0], y[1], y[2])
+    sqrt(2)*Sum((1 - sqrt(2)/2)**n*exp((n + 1)*(y_1 + y_2 + y_3) - exp(y_1) -
+    exp(y_2) - exp(y_3))/gamma(n + 1)**3, (n, 0, oo))/2
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Generalized_multivariate_log-gamma_distribution
+    .. [2] https://www.researchgate.net/publication/234137346_On_a_multivariate_log-gamma_distribution_and_the_use_of_the_distribution_in_the_Bayesian_analysis
+
+    """
+    return multivariate_rv(GeneralizedMultivariateLogGammaDistribution,
+                            syms, omega, v, l, mu)
+
+GMVLG = GeneralizedMultivariateLogGamma
```

Comment:
Looking up GMVLG does not give anything, though Wikipedia does use the acronym "G-MVLG". Can we stick with the complete name?

Reply:
We can stick with complete name. No issues in that, I just thought that it will help the person who is using it somewhere in `sympy` and will make the code shorter. However, we can use `from import ... as ...`. I will update it.